src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -35,16 +35,19 @@ def _polish_bad_errors(ex):
         import json
         from knack.util import CLIError
         try:
-            detail = json.loads(ex.response.text)['Message']
-            if creating_plan:
-                if 'Requested features are not supported in region' in detail:
-                    detail = ("Plan with linux worker is not supported in current region. For " +
-                              "supported regions, please refer to https://docs.microsoft.com/en-us/"
-                              "azure/app-service-web/app-service-linux-intro")
-                elif 'Not enough available reserved instance servers to satisfy' in detail:
-                    detail = ("Plan with Linux worker can only be created in a group " +
-                              "which has never contained a Windows worker, and vice versa. " +
-                              "Please use a new resource group. Original error:" + detail)
+            if 'text/plain' in ex.response.headers['Content-Type']:  # HTML response
```

Comment:
Instead of this, how about this:
```Python
try:
    details = json.loads(ex.response.text)['Message']
    # remaining logic
except json.JSONDecodeError:
    details = ex.response.txt
```

Reply:
- https://docs.python.org/3/library/json.html#json.loads
- https://docs.python.org/2.7/library/json.html#json.loads